### ISSUE TITLE
t440s Desktop not logging in after upgrading to 22.04

### ISSUE THREAD

**Issue by 130s** [kinu-garage/hut_10sqft#975]
After https://github.com/kinu-garage/hut_10sqft/issues/974 t440s is upgraded to Ubuntu 22.04 then after reboot, desktop doesn't let the user log in and just comes back to the same login window.

```
sudo apt update && apt upgrade
sudo do-release-upgrade
reboot
```

**Comment by 130s** [comment]
<details><summary>Btw, I reverted a config that Dropbox may need.</summary>

```
:

Configuration file '/etc/sysctl.conf'
 ==> Modified (by you or by a script) since installation.
 ==> Package distributor has shipped an updated version.
   What would you like to do about it ?  Your options are:
    Y or I  : install the package maintainer's version
    N or O  : keep your currently-installed version
      D     : show the differences between the versions
      Z     : start a shell to examine the situation
 The default action is to keep your current version.
*** sysctl.conf (Y/I/N/O/D/Z) [default=N] ? D
--- /etc/sysctl.conf    2020-11-24 15:45:39.635145839 -0500
+++ /etc/sysctl.conf.dpkg-new   2022-02-25 06:32:20.000000000 -0500
@@ -9,7 +9,7 @@
 # Uncomment the following to stop low-level messages on console
 #kernel.printk = 3 4 1 3
 
-##############################################################3
+###################################################################
 # Functions previously found in netbase
 #
 
@@ -66,4 +66,3 @@
 # for what other values do
 #kernel.sysrq=438
 
-fs.inotify.max_user_watches=1000000

Configuration file '/etc/sysctl.conf'
 ==> Modified (by you or by a script) since installation.
 ==> Package distributor has shipped an updated version.
   What would you like to do about it ?  Your options are:
    Y or I  : install the package maintainer's version
    N or O  : keep your currently-installed version
      D     : show the differences between the versions
      Z     : start a shell to examine the situation
 The default action is to keep your current version.
```
</details>

**Comment by 130s** [comment]
<details><summary>From syslog</summary>

```
Jan 12 15:54:54 130s-t440s goa-daemon[3145]: secret_password_lookup_sync() returned NULL
Jan 12 15:54:54 130s-t440s logrotate[399407]: error: state file /var/lib/logrotate/status is world-readable and thus can be locked from other unprivileged \
users. Skipping lock acquisition...
Jan 12 15:54:55 130s-t440s dbus-daemon[932]: [system] Activating via systemd: service name='net.reactivated.Fprint' unit='fprintd.service' requested by ':1\
.5102' (uid=0 pid=399549 comm="gdm-session-worker [pam/gdm-fingerprint] " label="unconfined")
Jan 12 15:54:55 130s-t440s systemd[1]: Starting Fingerprint Authentication Daemon...
Jan 12 15:54:55 130s-t440s systemd[1]: Stopping Make remote CUPS printers available locally...
Jan 12 15:54:55 130s-t440s systemd[1]: cups-browsed.service: Deactivated successfully.
Jan 12 15:54:55 130s-t440s systemd[1]: Stopped Make remote CUPS printers available locally.
Jan 12 15:54:55 130s-t440s systemd[1]: Stopping CUPS Scheduler...
Jan 12 15:54:55 130s-t440s fprintd[399555]: Failed to install a sleep delay inhibitor: GDBus.Error:org.freedesktop.login1.OperationInProgress: The operatio\
n inhibition has been requested for is already running
Jan 12 15:54:55 130s-t440s dbus-daemon[932]: [system] Successfully activated service 'net.reactivated.Fprint'
Jan 12 15:54:55 130s-t440s systemd[1]: Started Fingerprint Authentication Daemon.
Jan 12 15:54:55 130s-t440s systemd[1]: cups.service: Deactivated successfully.
Jan 12 15:54:55 130s-t440s systemd[1]: Stopped CUPS Scheduler.
Jan 12 15:54:55 130s-t440s systemd[1]: cups.path: Deactivated successfully.
Jan 12 15:54:55 130s-t440s systemd[1]: Stopped CUPS Scheduler.
Jan 12 15:54:55 130s-t440s systemd[1]: Stopping CUPS Scheduler...
Jan 12 15:54:55 130s-t440s systemd[1]: Started CUPS Scheduler.

:

Jan 12 15:54:55 130s-t440s gsd-color[1564]: failed to connect to device: Failed to connect to missing device /org/freedesktop/ColorManager/devices/cups_Bro\
ther_MFC_L2750DW_series
Jan 12 15:54:55 130s-t440s systemd[1]: Starting CUPS Scheduler...
Jan 12 15:54:55 130s-t440s systemd[1]: Started CUPS Scheduler.
Jan 12 15:54:55 130s-t440s kernel: [11658.716087] audit: type=1400 audit(1705092895.699:100): apparmor="DENIED" operation="capable" profile="/usr/sbin/cups\
d" pid=399588 comm="cupsd" capability=12  capname="net_admin"
Jan 12 15:54:55 130s-t440s systemd[1]: Started Make remote CUPS printers available locally.
Jan 12 15:54:55 130s-t440s kernel: [11658.742862] audit: type=1400 audit(1705092895.727:101): apparmor="DENIED" operation="capable" profile="/usr/sbin/cups\
-browsed" pid=399590 comm="cups-browsed" capability=23  capname="sys_nice"
Jan 12 15:54:55 130s-t440s systemd[2869]: snap.snapd-desktop-integration.snapd-desktop-integration.service: Scheduled restart job, restart counter is at 49\
51.
Jan 12 15:54:55 130s-t440s systemd[2869]: Stopped Service for snap application snapd-desktop-integration.snapd-desktop-integration.
Jan 12 15:54:55 130s-t440s systemd[2869]: Started Service for snap application snapd-desktop-integration.snapd-desktop-integration.
Jan 12 15:54:55 130s-t440s systemd[1]: logrotate.service: Deactivated successfully.
Jan 12 15:54:55 130s-t440s snapd-desktop-i[399669]: Not loading module "atk-bridge": The functionality is provided by GTK natively. Please try to not load \
it.
Jan 12 15:54:55 130s-t440s snapd-desktop-i[399669]: Failed to do gtk init. Waiting for a new session with desktop capabilities.
Jan 12 15:54:55 130s-t440s snapd-desktop-i[399669]: Checking session /org/freedesktop/login1/session/_32...
Jan 12 15:54:55 130s-t440s snapd-desktop-i[399669]: Checking session /org/freedesktop/login1/session/c1...
Jan 12 15:54:55 130s-t440s snapd-desktop-i[399669]: Checking session /org/freedesktop/login1/session/_35...
Jan 12 15:54:55 130s-t440s snapd-desktop-i[399669]: Is a desktop session! Forcing a reload.
Jan 12 15:54:55 130s-t440s snapd-desktop-i[399669]: Checking session /org/freedesktop/login1/session/_34...
Jan 12 15:54:55 130s-t440s snapd-desktop-i[399669]: Is a desktop session! Forcing a reload.
Jan 12 15:54:55 130s-t440s snapd-desktop-i[399669]: Loop exited. Forcing reload.
Jan 12 15:54:56 130s-t440s systemd[1]: dpkg-db-backup.service: Deactivated successfully.
Jan 12 15:54:56 130s-t440s systemd[1]: Finished Daily dpkg database backup service.
Jan 12 15:54:57 130s-t440s systemd[2869]: snap.snapd-desktop-integration.snapd-desktop-integration.service: Scheduled restart job, restart counter is at 49\
52.
Jan 12 15:54:57 130s-t440s systemd[2869]: Stopped Service for snap application snapd-desktop-integration.snapd-desktop-integration.
Jan 12 15:54:57 130s-t440s systemd[2869]: Started Service for snap application snapd-desktop-integration.snapd-desktop-integration.
Jan 12 15:54:58 130s-t440s snapd-desktop-i[399764]: Not loading module "atk-bridge": The functionality is provided by GTK natively. Please try to not load \
it.
Jan 12 15:54:58 130s-t440s snapd-desktop-i[399764]: Failed to do gtk init. Waiting for a new session with desktop capabilities.
Jan 12 15:54:58 130s-t440s snapd-desktop-i[399764]: Checking session /org/freedesktop/login1/session/_32...
Jan 12 15:54:58 130s-t440s snapd-desktop-i[399764]: Checking session /org/freedesktop/login1/session/c1...
Jan 12 15:54:58 130s-t440s snapd-desktop-i[399764]: Checking session /org/freedesktop/login1/session/_35...
Jan 12 15:54:58 130s-t440s snapd-desktop-i[399764]: Is a desktop session! Forcing a reload.
Jan 12 15:54:58 130s-t440s snapd-desktop-i[399764]: Checking session /org/freedesktop/login1/session/_34...
Jan 12 15:54:58 130s-t440s snapd-desktop-i[399764]: Is a desktop session! Forcing a reload.
Jan 12 15:54:58 130s-t440s snapd-desktop-i[399764]: Loop exited. Forcing reload.
Jan 12 15:54:58 130s-t440s systemd[1]: systemd-rfkill.service: Deactivated successfully.
Jan 12 15:55:00 130s-t440s systemd[1]: NetworkManager-dispatcher.service: Deactivated successfully.
Jan 12 15:55:00 130s-t440s systemd[2869]: snap.snapd-desktop-integration.snapd-desktop-integration.service: Scheduled restart job, restart counter is at 49\
53.
Jan 12 15:55:00 130s-t440s systemd[2869]: Stopped Service for snap application snapd-desktop-integration.snapd-desktop-integration.
Jan 12 15:55:00 130s-t440s systemd[2869]: Started Service for snap application snapd-desktop-integration.snapd-desktop-integration.
Jan 12 15:55:00 130s-t440s snapd-desktop-i[399842]: Not loading module "atk-bridge": The functionality is provided by GTK natively. Please try to not load \
it.
Jan 12 15:55:00 130s-t440s snapd-desktop-i[399842]: Failed to do gtk init. Waiting for a new session with desktop capabilities.
Jan 12 15:55:00 130s-t440s snapd-desktop-i[399842]: Checking session /org/freedesktop/login1/session/_32...
Jan 12 15:55:00 130s-t440s snapd-desktop-i[399842]: Checking session /org/freedesktop/login1/session/c1...
Jan 12 15:55:00 130s-t440s snapd-desktop-i[399842]: Checking session /org/freedesktop/login1/session/_35...
Jan 12 15:55:00 130s-t440s snapd-desktop-i[399842]: Is a desktop session! Forcing a reload.
Jan 12 15:55:00 130s-t440s snapd-desktop-i[399842]: Checking session /org/freedesktop/login1/session/_34...
Jan 12 15:55:00 130s-t440s snapd-desktop-i[399842]: Is a desktop session! Forcing a reload.
Jan 12 15:55:00 130s-t440s snapd-desktop-i[399842]: Is a desktop session! Forcing a reload.
Jan 12 15:55:00 130s-t440s snapd-desktop-i[399842]: Loop exited. Forcing reload.
Jan 12 15:55:01 130s-t440s systemd[1]: Started Session 13 of User n130s.
Jan 12 15:55:01 130s-t440s kernel: [11664.319720] rfkill: input handler enabled
Jan 12 15:55:01 130s-t440s gnome-session-c[399896]: cannot open display: :0
Jan 12 15:55:01 130s-t440s gnome-session-c[399897]: cannot open display: :0
Jan 12 15:55:01 130s-t440s gnome-session[399851]: gnome-session-binary[399851]: WARNING: software acceleration check failed: Child process exited with code\
 1
Jan 12 15:55:01 130s-t440s gnome-session-binary[399851]: WARNING: software acceleration check failed: Child process exited with code 1
Jan 12 15:55:01 130s-t440s gnome-session-f[399898]: Cannot open display:
Jan 12 15:55:01 130s-t440s gdm3: Gdm: GdmDisplay: Session never registered, failing
Jan 12 15:55:01 130s-t440s kernel: [11664.501481] rfkill: input handler disabled
Jan 12 15:55:02 130s-t440s org.gnome.Shell.desktop[399906]: The XKEYBOARD keymap compiler (xkbcomp) reports:
Jan 12 15:55:02 130s-t440s org.gnome.Shell.desktop[399906]: > Warning:          Unsupported maximum keycode 708, clipping.
Jan 12 15:55:02 130s-t440s org.gnome.Shell.desktop[399906]: >                   X11 cannot support keycodes above 255.
Jan 12 15:55:02 130s-t440s org.gnome.Shell.desktop[399906]: Errors from xkbcomp are not fatal to the X server
Jan 12 15:55:02 130s-t440s gnome-shell[1399]: Object .MetaInputDeviceNative (0x7f36e40bd6b0), has been already disposed — impossible to access it. This mi\
ght be caused by the object having been destroyed from C code using something such as destroy(), dispose(), or remove() vfuncs.
Jan 12 15:55:02 130s-t440s org.gnome.Shell.desktop[1399]: == Stack trace for context 0x55a28f1654a0 ==
Jan 12 15:55:02 130s-t440s org.gnome.Shell.desktop[1399]: #0   55a291289c60 i   resource:///org/gnome/shell/ui/keyboard.js:1231 (2801385a8790 @ 56)
Jan 12 15:55:02 130s-t440s org.gnome.Shell.desktop[1399]: #1   55a291289bb8 i   resource:///org/gnome/shell/ui/keyboard.js:1237 (2801385a87e0 @ 94)
Jan 12 15:55:02 130s-t440s org.gnome.Shell.desktop[1399]: #2   7ffd492034e0 b   self-hosted:1178 (2f3ca0db0a10 @ 454)
Jan 12 15:55:02 130s-t440s systemd[2869]: snap.snapd-desktop-integration.snapd-desktop-integration.service: Scheduled restart job, restart counter is at 49\
54.
Jan 12 15:55:02 130s-t440s systemd[2869]: Stopped Service for snap application snapd-desktop-integration.snapd-desktop-integration.
Jan 12 15:55:02 130s-t440s systemd[2869]: Started Service for snap application snapd-desktop-integration.snapd-desktop-integration.
Jan 12 15:55:02 130s-t440s snapd-desktop-i[399982]: Not loading module "atk-bridge": The functionality is provided by GTK natively. Please try to not load \
it.
Jan 12 15:55:02 130s-t440s snapd-desktop-i[399982]: Failed to do gtk init. Waiting for a new session with desktop capabilities.
Jan 12 15:55:02 130s-t440s snapd-desktop-i[399982]: Checking session /org/freedesktop/login1/session/_32...
Jan 12 15:55:02 130s-t440s snapd-desktop-i[399982]: Checking session /org/freedesktop/login1/session/c1...
Jan 12 15:55:02 130s-t440s snapd-desktop-i[399982]: Checking session /org/freedesktop/login1/session/_35...
Jan 12 15:55:02 130s-t440s snapd-desktop-i[399982]: Is a desktop session! Forcing a reload.
Jan 12 15:55:02 130s-t440s snapd-desktop-i[399982]: Checking session /org/freedesktop/login1/session/_34...
Jan 12 15:55:02 130s-t440s snapd-desktop-i[399982]: Is a desktop session! Forcing a reload.
Jan 12 15:55:02 130s-t440s snapd-desktop-i[399982]: Checking session /org/freedesktop/login1/session/_313...
Jan 12 15:55:02 130s-t440s snapd-desktop-i[399982]: Is a desktop session! Forcing a reload.
Jan 12 15:55:02 130s-t440s snapd-desktop-i[399982]: Loop exited. Forcing reload.

:

Jan 12 15:54:55 130s-t440s systemd[2869]: Stopped Service for snap application snapd-desktop-integration.snapd-desktop-integration.
Jan 12 15:54:55 130s-t440s systemd[2869]: Started Service for snap application snapd-desktop-integration.snapd-desktop-integration.
Jan 12 15:54:55 130s-t440s systemd[1]: logrotate.service: Deactivated successfully.
Jan 12 15:54:55 130s-t440s systemd[1]: Finished Rotate log files.
Jan 12 15:54:55 130s-t440s snapd-desktop-i[399669]: Not loading module "atk-bridge": The functionality is provided by GTK natively. Please try to not load \
it.
Jan 12 15:54:55 130s-t440s snapd-desktop-i[399669]: Failed to do gtk init. Waiting for a new session with desktop capabilities.
Jan 12 15:54:55 130s-t440s snapd-desktop-i[399669]: Checking session /org/freedesktop/login1/session/_32...
Jan 12 15:54:55 130s-t440s snapd-desktop-i[399669]: Checking session /org/freedesktop/login1/session/c1...
Jan 12 15:54:55 130s-t440s snapd-desktop-i[399669]: Checking session /org/freedesktop/login1/session/_35...
Jan 12 15:54:55 130s-t440s snapd-desktop-i[399669]: Is a desktop session! Forcing a reload.
Jan 12 15:54:55 130s-t440s snapd-desktop-i[399669]: Checking session /org/freedesktop/login1/session/_34...
Jan 12 15:54:55 130s-t440s snapd-desktop-i[399669]: Is a desktop session! Forcing a reload.
Jan 12 15:54:55 130s-t440s snapd-desktop-i[399669]: Loop exited. Forcing reload.
```
</details>

**Comment by 130s** [comment]
Similar issue occurred on p16s and resolved by `bash` update https://github.com/kinu-garage/hut_10sqft/issues/804#issuecomment-1484014667. So trying to update `hut_10sqft` repo.
```
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights and the repository exists.
```
I found on my github account there's no longer an ssh key for this host so I recreated it `ssh-keygen` and uploaded the generated `.pub` content. Then `git` is now able to communicate with the remote.

Updating the `hut_10sqft` repo to date resolved the issue.
